src/dispatch/auth/views.py

```diff
@@ -134,32 +122,21 @@ def login_user(
     organization: OrganizationSlug,
     db_session: Session = Depends(get_db),
 ):
-    org = organization_service.get_by_slug(db_session=db_session, slug=organization)
-    if not org:
-        raise HTTPException(
-            status_code=status.HTTP_422_UNPROCESSABLE_ENTITY,
-            detail=[
-                {
-                    "msg": "Organization not found.",
-                    "loc": ["organization"],
-                    "type": "Not found",
-                }
-            ],
-        )
-
     user = get_by_email(db_session=db_session, email=user_in.email)
     if user and user.check_password(user_in.password):
         return {"token": user.token}
 
-    raise HTTPException(
-        status_code=status.HTTP_422_UNPROCESSABLE_ENTITY,
-        detail=[
-            {
-                "msg": "Invalid username or password",
-                "loc": ["username"],
-                "type": "BadUsernamePassword",
-            }
-        ],
+    raise ValidationError(
+        [
+            ErrorWrapper(
+                InvalidValueError(msg="Invalid username."),
+                loc="username",
+            ),
+            ErrorWrapper(
+                InvalidValueError(msg="Invalid password."),
+                loc="password",
+            ),
+        ]
```

Comment:
My preference would be to say something like "You have entered an invalid username or password" instead of letting a potential malicious user know which one they got wrong.

Reply:
The error returns both, regardless of which was actually the issue.

Reply:
So, for example, if login auth fails we show "Invalid username. Invalid password" in the UI?

Reply:
The API will return both, Yes. The error response is a list. We haven't traditionally displayed those very well (or at all in some cases). My hope is that with this change the error response will be more uniform across the API endpoints and we can display them better in the UI.